### ISSUE TITLE
--notstarted shows events that have started and finished

### ISSUE THREAD

**Issue by hobarrera** [pimutils/khal#690]
```
$ date                                      
Wed Jul 26 12:58:58 -03 2017 
$ khal list --notstarted | head -n 2        
Today, Wed 2017-07-26                       
12:15-12:45 Reserved for Lunch ⟳  
```

Not sure if it being recurring might be relevant or not, but it's worth mentioning. Also, the event is in a different timezone (but the displayed date is correctly converted, of course).

**Comment by geier** [comment]
Note that the default for `list` is `today 00:00 + defaulttimedelta` not `now + defaulttimedelta` so this works as intended. We can however discuss how we can make this more intuitive.

**Comment by hobarrera** [comment]
But I'm using `--notstarted`, which says `Print only events that have not started.`.

**Comment by geier** [comment]
That haven't started before the queried time range, which starts at
00:00 today. Try `khal list --notstarted now`.

Quoting Hugo Osvaldo Barrera (2017-07-26 18:28:33)

**Comment by hobarrera** [comment]
Uh, cool, the keyword `now` works for me. FWIW, I haven't been able to get any date to parse, even when explicitly setting the format to it:

```
$ khal list $(date +%Y%m%d%H%M)
[...]
ValueError: Could not parse `('201707290121',)` as a daterange
```

```
$ grep datetimeformat .config/khal/config 
datetimeformat = %Y%m%d%H%M
```

I also tried a bunch of other different formats (both when specifying them in the config, as well as not specifying any in the config). I couldn't get any matches, even for standard things like `2017-07-29` (with or without hour). Maybe something like [`parsedatetime`](https://pypi.python.org/pypi/parsedatetime) might be of use here?

As far as my initial issue, using `now` works (I just wanted to get the first next event 🙂 ).

**Comment by geier** [comment]
You will also need to set longdatetimeformat. Perhaps we should make it more intuitive by renaming datetimeformat -> shortdatetimeformat and longdatetimeformat -> datetimeformat.

**Comment by hobarrera** [comment]
TBH, I think the best "fix" is to update the output of `khal list --help`, so that it states that it uses "longdatetimeformat". This would pretty much make it fool-proof. 🙂 

Another suggestion (but this requires more work), is to try different formats in order (eg: `longdatetimeformat`, then  `datetimeformat`, and finally, maybe `parsedatetime`)

**Comment by hobarrera** [comment]
Oh, thanks, BTW! ☺️

**Comment by geier** [comment]
it should use datetimeformat as well, I messed that up when we changed
the defaults and told our parser to ignore datetimeformat when it
contains the year.

Quoting Hugo Osvaldo Barrera (2017-07-31 16:05:16)

**Comment by geier** [comment]
fixed with a30a4d9
